charts/seed-operatingsystemconfig/original/templates/kubelet/_kubelet.flags

```diff
@@ -30,6 +29,10 @@
 {{- if eq .Values.worker.cri.name .Values.osc.cri.names.containerd }}
 --container-runtime=remote
 --container-runtime-endpoint=unix:///run/containerd/containerd.sock
+{{- else }}
+--pod-infra-container-image={{ index .Values.images "pause-container" }}
 {{- end -}}
+{{- else }}
+--pod-infra-container-image={{ index .Values.images "pause-container" }}
```

Comment:
Why does this flag exist twice now?

Reply:
Because there are 2 else conditions...
not `.Values.worker.cri` 
not ` eq .Values.worker.cri.name .Values.osc.cri.names.containerd`

Hope I didn't make mistake.

Reply:
Ah, I see now, thanks. Looks correct... but we should definitely refactor these charts as well in the future.

Reply:
BTW, do we have plan to make next release based on master? Otherwise, I need to cherry pick to release branck.

Reply:
Yes, but it will take another week or so. Feel free to open a PR to the release branch once this PR is merged to master.